### DISCUSSION TITLE
Payment Gateway

### DISCUSSION THREAD

**Discussion by CentaurId** [deaafrizal/forum#14] [Troubleshooting Room]
Selamat sore bang tadi gue barusan nyoba video paymeny gateway dan ada pas click checkout itu error seperti dibawah. itu kenapa ya erornya ?
<img width="919" alt="payment gateway" src="https://github.com/deaaprizal/forum/assets/119429252/c38f94e6-8153-4b37-9308-4060554880a6">
<img width="952" alt="payment gatewayy" src="https://github.com/deaaprizal/forum/assets/119429252/97d2f79e-2fd1-4bbf-9a54-d36ecb1a80a7">

**Comment by deaafrizal** [comment]
coba liat file route.js baris ke 16 di folder api bro.

**Reply by CentaurId** [reply]
![route payment](https://github.com/deaaprizal/forum/assets/119429252/b54d64ac-518a-4a1e-81c9-68938c326ce5)

**Reply by deaafrizal** [reply]
sintaksnya sih udah sama, coba lu ganti dlu isi route.js nya pake ini:

`export async function GET(request) {
    return NextResponse.json({test: "OK"})
}`

atau ini

`export async function GET(request) {
    return Response.json({test: "OK"})
}`


kalau udah save, cek di url browser ke /api/token/

muncul gak route api nya.

**Reply by CentaurId** [reply]
![lanjutan payment](https://github.com/deaaprizal/forum/assets/119429252/198937f5-0b13-4f66-b570-c28724de4bee)
<img width="952" alt="cek22" src="https://github.com/deaaprizal/forum/assets/119429252/44e61169-db85-4a4a-81f8-4ac00ee0befc">
btw masih belum connect bang, dari route ke nextjsnya. itu gmana ya solusinya ? tapi kalo buat yang payment by link sudah bisa

**Reply by deaafrizal** [reply]
itu lu buka url nya apaan anjir, kok kebalik /token/api?
bukannya /api/token?
